chain/client/tests/process_blocks.rs

```diff
@@ -85,6 +85,7 @@ pub fn create_nightshade_runtimes(genesis: &Genesis, n: usize) -> Vec<Arc<dyn Ru
                 vec![],
                 vec![],
                 None,
+                None,
```

Comment:
Seems signature changes like that one could be very broad. Could we agree that instead we pass an aggregating structure containing fields instead of long arg list?

Reply:
Yes, I have considered doing that. I still might but I don’t want to include such refactoring in this commit since it’s an unrelated change.

Reply:
Note however that introducing a structure with arguments won’t fix the issue of changes to the structure affecting all call sites. It will make the calls more readable though thanks to the use of named fields. Issue is that some of the arguments are required and thus there won’t be any sensible implementation of Default for such structure. Builder would be the only solution here but that’s even larger refactoring.